### DISCUSSION TITLE
the office documents is really difficult to read

### DISCUSSION THREAD

**Discussion by huaiguoguo** [fastapi/sqlmodel#607] [Show and tell]
the office documents is really difficult to read  !!!!

**Comment by gamesbook** [comment]
Can you provide a link or example?

**Reply by huaiguoguo** [reply]
sanic.dev

**Comment by huaiguoguo** [comment]
> Can you provide a link or example?

sanic.dev

**Comment by jerivas** [comment]
I think you should report this to the Sanic repository: https://github.com/sanic-org/sanic

**Reply by huaiguoguo** [reply]
I misunderstood and expressed incorrectly, which led to your incorrect answers

i am meaning is : "all document of sqlmodel is really difficult to read"
i can't found the example of i want
